src/patch_helper.py

```diff
@@ -111,6 +111,14 @@ def trigger_enhanced_scan_patching(image_uri, patch_details_path, python_version
     :return: str, Returns constants.SUCCESS to allow the multi-threaded caller to know that the method has succeeded.
     """
     impacted_packages = get_impacted_os_packages(image_uri=image_uri, python_version=python_version)
+
+    ## TODO: Make this more scalable in future
```

Comment:
Improve this comment - add explanation for why this package needs to be added

Reply:
Added the detailed comment, thanks!